package.json

```diff
@@ -5,12 +5,13 @@
   "scripts": {
     "start": "node server/index.js",
     "server": "nodemon server/index.js",
-    "client": "cd client && npm start",
+    "client": " cd client  && npm run build",
```

Comment:
I think that our client script should stay as ```cd client && npm start```
if you'd like to add a client build script for heroku, I think that maybe it should be done in a separate script, say
```"client:build" : "cd client  && npm run build"```

thank you!!
💓